angr/knowledge_plugins/indirect_jumps.py

```diff
@@ -6,13 +6,27 @@ class IndirectJumps(KnowledgeBasePlugin, dict):
     def __init__(self, kb):
         super(IndirectJumps, self).__init__()
         self._kb = kb
-        self.resolved = set()
         self.unresolved = set()
 
+        # dict format: {indirect_address: [resolved_addresses]}
+        self.resolved = {}
+
     def copy(self):
         o = IndirectJumps(self._kb)
-        o.resolved.update(self.resolved)
         o.unresolved.update(self.unresolved)
+        o.resolved = {}
+        for k, v in self.resolved.items():
+            o.resolved[k] = v
+
+    def update_resolved_addrs(self, indirect_address: int, resolved_addresses: list):
```

Comment:
And you may want to use `typing.List` here, which is subscriptable. For example:
```py
from typing import Dict, List
def my_func(input: List[int]) -> Dict[int, int]:
    return {}
```

Reply:
fixed :)